src/scripts/router.js

```diff
@@ -93,16 +20,28 @@ function initRouter(profil, stockageLocal) {
     router.hooks({
         before: function (done) {
             // Global hook to redirect on the correct page given registered data.
-            var requestedPage = getCurrentPageName() || 'introduction'
-            var redirectedPage = redirectToUnansweredQuestions(requestedPage, profil)
+            var requestedPage = pagination.getCurrentPageName() || 'introduction'
+            var redirectedPage = pagination.redirectToUnansweredQuestions(
+                requestedPage,
+                app.profil
+            )
             if (redirectedPage) {
                 router.navigate(redirectedPage)
             }
+            var header = document.querySelector('header section')
+            if (!app.profil.nom) {
```

Comment:
```suggestion
            if (typeof app.profil.nom === 'undefined') {
```

I guess we have to be more robust on that one, except if Parcel converts the assertion? 🤔 If I remember correctly IE11 is a bit touchy on that kind of comparison…